celery/utils/dispatch/weakref_backports.py

```diff
@@ -28,7 +28,7 @@ def __new__(cls, meth, callback=None):
             func = meth.__func__
         except AttributeError:
             raise TypeError(
-                "Argument should be a bound method, not {0}".format(
+                "Argument should be a bound method, not {}".format(
```

Comment:
f

Reply:
That's a backport. I don't think we need it.

Reply:
right